### PR TITLE
Dehydrator/Crystallizer Emissive Textures

### PR DESCRIPTION
**Pull request by jude123412**
Added Emissive textures for Dehydrator and Crystallizer
Changes to Crystallizer overlay_front, overlay_front_emissive, overlay_front_active and overlay_front_active_emissive textures
Added Textures to the side of the Dehydrator and Crystallizer

### CODE REVIEW

File: src/main/resources/assets/gregtech/textures/blocks/machines/crystallizer/overlay_front_emissive.png
Comment (tekcay) [inline src/main/resources/assets/gregtech/textures/blocks/machines/crystallizer/overlay_front_emissive.png:1]: Should be red, plus is it necessary to have an emissive texture for a non active machine ?

File: src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png
Comment (tekcay) [inline src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png:1]: has to be emissive ?

**Review by BL00DKNIGHT — APPROVED** [review]
Looks ok

File: src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_top_active.png.mcmeta
Comment (tekcay) [inline src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_top_active.png.mcmeta:1]: it is still not working on my side, can you try with the tungsten fix branch to see if it works ?

**Review by tekcay — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png
Comment (jude123412) [reply src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png:1]: I doesn't have to be emissive but including it means that the config for emissive textures still works when enabled

File: src/main/resources/assets/gregtech/textures/blocks/machines/crystallizer/overlay_front_emissive.png
Comment (BLocky104) [reply src/main/resources/assets/gregtech/textures/blocks/machines/crystallizer/overlay_front_emissive.png:1]: I think we should keep the emissive part, its a really good part of GTCEu so why not use it

File: src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png
Comment (tekcay) [reply src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png:1]: yeah, but usually machine don't have bloom when they are idle, which is the case here, that's why I was asking

Comment (tekcay) [reply src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_front_emissive.png:1]: oh my bad saw your comment above

File: src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_top_active.png.mcmeta
Comment (jude123412) [reply src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_top_active.png.mcmeta:1]: I have checked with latest version and all textures are working properly :)

Comment (tekcay) [reply src/main/resources/assets/gregtech/textures/blocks/machines/dehydrator/overlay_top_active.png.mcmeta:1]: fine then :)

**Review by tekcay — APPROVED** [review]

**Pull request merged**